field/TinyLang/Field/Core.hs

```diff
@@ -80,7 +84,7 @@ instance Field f => Field (Expr f (AField f)) where
     inv = EAppUnOp Inv
     mul = EAppBinOp Mul
     div = EAppBinOp Div
-
+                                              
```

Comment:
Not too important, but it's generally good to configure your editor to remove trailing whitespace, so that it doesn't end up in diffs.

Reply:
Strange.  I've got emacs using stylish-haskell when I save and usually it gets rid of trailing whitespace.